Core/MIPS/MIPSVFPUUtils.cpp

```diff
@@ -946,34 +936,182 @@ void vfpu_sincos_single(float angle, float &sine, float &cosine) {
 	}
 }
 
-float vfpu_sin_double(float angle) {
-	return (float)sin((double)angle * M_PI_2);
+float vfpu_sin_mod2(float a) {
```

Comment:
wouldn't mod4 be a more accurate name?

Reply:
First step is mod by 4 (since it has a repeating pattern by 4), but then it mods by 2 right below that (which may negate the result or negate the input.)  Ultimately, as in the other note, I may want to mod by 1 and swap sin/cos (which I was already doing in my CORDIC test code), but that part adds unnecessary extra instructions without much accuracy benefit at this point.

-[Unknown]